auth/database/CRUDController.py

```diff
@@ -9,223 +9,280 @@
 from database.Models import Permission, User, Group, PermissionEnum
 from flaskAlchemyInit import HTTPRequestError
 
-#helper function to check user fields
-def checkUser(user, ignore = []):
+
+# Helper function to check user fields
+def checkUser(user, ignore=[]):
```

Comment:
One can also use tags to tell devs expected parameters, returned values, thrown exceptions.
It usually helps with automatic documentation generation, as well as linting and autocompletion tools.

[This](https://stackoverflow.com/a/24385103) lists a couple that may be used. I for one like reST  more - it resembles javadoc/doxygen tags.